playbook.md

```diff
@@ -2318,3 +2318,12 @@ The process for writing a new post is:
 ### Twitter
 
 ### Facebook
+
+## Sustainability 
+We at dxw are working towards becoming a more environmentally sustainable company. We are focusing on three main areas to reduce our impact; 
```

Comment:
Did you mean to add more after the semicolon? If not, it should either be a colon or a period.